src/plugins/prefixed.js

```diff
@@ -5,7 +5,8 @@ import pascalize from '../utils/pascalize'
 export default {
   supportedProperty: (prop, style) => {
     const pascalized = pascalize(prop)
-    if (prop.substring(0, 2) === '--') return prop
+    // Return custom CSS variable or already prefixed value without prefixing.
+    if (prop[0] === '-' || prop[1] === '-') return prop
```

Comment:
lol, this is broken `a-`

Reply:
you need 2 cases
1. there is a single dash - it is already prefixed
2. there are 2 prefixes - it doesn't need prefixing

```
if (prop[0] === '-') return prop
if (prop[0] === '-' && prop[1] === '-' ) return prop
```

Reply:
actually if there is `prop[0] === '-'` why would it prefix when there are 2 dashes? lol this is some bs, this should already work, did you try it?

Reply:
That's an unreal case, but ok :D

Reply:
it seems we don't need to check on 2 dashes at all, first dash is all we need to bail out, but it's good to have a test for both -a and --a

Reply:
Actually same goes to checking seconds prefix. We don't need to check it at all, at least I don't know cases where it should be useful. I mean, if first char is `-` we already should return the property without prefixing.

Reply:
we don't actually need to check second prefix if we already exit on the first one, I didn't realize we don't do anythings special for the case of custom props

Reply:
Sorry, didn't see your comment. That's why I did check only with the first prefix only from start. Later removed it, because I think users don't need that. But autoprefixer doing that, so better to keep this feature.